### PR TITLE
Added php

### PR DESCRIPTION
**Pull request by Kibadda**
First I want to say great plugin. Been looking for a replacement for `splitjoin.vim` for a long time.

I added php as suggested in `tests/README.md`, but I can not get the tests to run without errors.
It looks like `split` and `join` are not executed on the given line (see attached log).

```
nvim --headless --noplugin -u tests/minimal.vim -c "PlenaryBustedDirectory tests/langs/php_spec.lua {minimal_init = 'tests/minimal.vim'}"
Starting...
Scheduling: tests/langs/php_spec.lua

========================================
Testing:        /home/michael/plugins/treesj/tests/langs/php_spec.lua
Fail    ||      TreeSJ SPLIT: lang "php", node "array_creation_expression", preset default
            ./tests/utils.lua:23: Expected objects to be the same.
            Passed in:
            (table: 0x7fd7f15aabe0) {
             *[1] = '$array = ['test' => 15, '12' => 'Hello', 'World' => '!'];'
              [2] = ''
              [3] = '// RESULT OF SPLIT (node "array_creation_expression", preset default)'
              [4] = '$array = ['
              [5] = '    'test' => 15,' }
            Expected:
            (table: 0x7fd7f1542a08) {
             *[1] = '$array = ['
              [2] = '    'test' => 15,'
              [3] = '    '12' => 'Hello','
              [4] = '    'World' => '!','
              [5] = '];' }

            stack traceback:
                ./tests/utils.lua:23: in function <./tests/utils.lua:15>

[TreeSJ]: Language "php" is not configured
Fail    ||      TreeSJ JOIN: lang "php", node "array_creation_expression", preset default
            ./tests/utils.lua:23: Expected objects to be the same.
            Passed in:
            (table: 0x7fd7f15861a0) {
             *[1] = '$array = [' }
            Expected:
            (table: 0x7fd7f15e4678) {
             *[1] = '$array = ['test' => 15, '12' => 'Hello', 'World' => '!'];' }

            stack traceback:
                ./tests/utils.lua:23: in function <./tests/utils.lua:15>


Success:        0
Failed :        2
Errors :        0
========================================
Tests Failed. Exit: 1
make: *** [Makefile:11: test-php] Error 1
```

Maybe I'm missing something. Do you have any idea?

### CODE REVIEW

**Comment by Wansmer** [comment]
Hi! Thanks for PR. 
I tried to run your test: 'join' are passed, split not passed because in test tab is 4 spaces, but in minimal.vim 2. When I removed extra spaces - both test are passed.

Unfortunately, I have no idea, why on your computer test are failed. Maybe, problem in loading plugin. Are you have treesj in 'packer' cache?
At packer should be `use('~/PATH_TO_LOCAL/treesj')` not `use('Wansmer/treesj')`
Снимок экрана 2022-11-24 в 12 44 39

**Comment by Kibadda** [comment]
Yeah, I was still using `use('Wansmer/treesj')`. Thanks :).
I'm gonna implement the other tests and mark as ready.

**Comment by Wansmer** [comment]
> I'm gonna implement the other tests and mark as ready.

Excellent! Please, update your 'main' before – I did some changes.

File: lua/treesj/langs/init.lua [hunk 1]
```diff
@@ -8,6 +8,8 @@ local langs = {
   'css',
   'scss',
   'tsx',
+  'jsx',
```
Comment (Wansmer) [inline lua/treesj/langs/init.lua]: Not needed, because treesitter recognize jsx as javascript.

**Review by Wansmer — COMMENTED** [review]
Thank for your work! Please, add php to list of preconfigured plugins in README.

File: lua/treesj/langs/init.lua (hunk 1 above)
Comment (Kibadda) [reply lua/treesj/langs/init.lua]: Yeah I saw that too and removed that in `fix failing tests`

**Comment by Kibadda** [comment]
Added php to the list in the README.

**Pull request merged**